easybuild/easyconfigs/g/gnuplot/gnuplot-5.0.1-intel-2015b.eb

```diff
@@ -0,0 +1,37 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg/Luxembourg Centre for Systems Biomedicine
+# Authors::   Fotis Georgatos <fotis.georgatos@uni.lu>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-97.html
+##
+easyblock = 'ConfigureMake'
+
+name = 'gnuplot'
+version = '5.0.1'
+
+homepage = 'http://gnuplot.sourceforge.net/'
+description = """Portable interactive, function plotting utility"""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [('http://sourceforge.net/projects/gnuplot/files', 'download')]
+
+dependencies = [
+    ('cairo', '1.14.4'),
+    ('libjpeg-turbo', '1.4.2'),
+    ('libgd', '2.1.1'),
```

Comment:
gnuplot's configure script doesn't check for libpng directly, so no need to add it as a direct dependency